src/AtmosphereModels/radiation_interface.jl

```diff
@@ -32,14 +34,79 @@ end
 """
 $(TYPEDSIGNATURES)
 
-Return a RadiativeTransferModel on `grid` with thermodynamic `constants` and using
-the `optical_thickness` model for radiative transfer.
+Construct a `RadiativeTransferModel` on `grid` using the specified `optics_flavor`.
+
+Valid optics flavors are:
+- `:gray` - Gray atmosphere radiation (O'Gorman & Schneider 2008)
+- `:clear_sky` - Full-spectrum clear-sky radiation using RRTMGP gas optics
+
+The `constants` argument provides physical constants for the radiative transfer solver.
+
+# Example
+
+```julia
+rtm = RadiativeTransferModel(grid, :gray, constants;
+    surface_temperature = 300,
+    surface_albedo = 0.1)
+
+rtm = RadiativeTransferModel(grid, :clear_sky, constants;
+    surface_temperature = 300,
+    surface_albedo = 0.1,
+    background_atmosphere = BackgroundAtmosphere(CO₂ = 400e-6))
+```
 """
-function RadiativeTransferModel(grid, constants, optical_thickness; kw...)
-    msg = "At the moment, RadiativeTransferModel requires RRTMGP and is only valid with
-          optical_thickness = GrayOpticalThicknessOGorman2008(FT). Received $optical_thickness."
-    throw(ArgumentError(msg))
-    return nothing
+function RadiativeTransferModel(grid::AbstractGrid, optics_flavor::Symbol, args...; kw...)
+    RadiativeTransferModel(grid, Val(optics_flavor), args...; kw...)
```

Comment:
There's no (non-erroring) method taking keyword arguments

Reply:
Ah, no, wait, it's in `ext/BreezeRRTMGPExt/clear_sky_radiative_transfer_model.jl`.  Uhm... 🤔

Reply:
Ah, I got it: 0e97bafda1530937832b1c514f726e2acd068f9b